server/views/taskList/taskListTemplate.pug

```diff
@@ -23,9 +23,20 @@ block content
   else if licenceStatus.decisions.unsuitableResult
     div.error-banner.marginTopMedium
       strong.bold The offender is presumed unsuitable for HDC release
-  else if user.role != 'DM' && licenceStatus.decisions.refused
+  else if user.role === 'CA' && licenceStatus.decisions.caRefused
     div.error-banner.marginTopMedium
-      strong.bold Home detention curfew refused - #{licenceStatus.decisions.refusalReason}
+      strong.bold Home detention curfew refused by Case Admin - #{licenceStatus.decisions.refusalReason}
+  else if user.role === 'CA' && licenceStatus.decisions.refused && licenceStatus.decisions.decisionComments
```

Comment:
Prefer the full name of the role to an abbreviated form: 'Home detention curfew refused by Prison Case Administrator'

Reply:
fair point
Done

Reply:
changed to 'prison case admin' and 'decision maker'